test/test_base.py

```diff
@@ -350,7 +374,7 @@ def test_scaled_mm_vs_emulated(self, base_dtype):
 
 
 class TestNumerics:
-    @pytest.mark.parametrize("float8_dtype", [torch.float8_e4m3fn, torch.float8_e5m2])
+    @pytest.mark.parametrize("float8_dtype", [fp8_e4m3_t(), fp8_e5m2_t()])
```

Comment:
I would recommend testing all cases on all hardware types instead.  For things not requiring a matmul, it should just work.  For things requiring a matmul, we have an emulation mode to at least help approximate it.